src/index.html

```diff
@@ -11,12 +11,113 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="stylesheet"
+      href="../../reset/reset.css"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <div>
+        <a
+          href="#"
+          class="logo"
+        >
+          <img
+            src="images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+      <div>
+        <nav class="nav">
+          <ul class="ul_list">
+            <li class="list">
+              <a
+                href="#apple"
+                class="links is-active"
+                id="apple"
```

Comment:
1. `id` is redundant
2. use nav children as its elements (class names)
3. `li` - is an item (not a list). `a` - is a single link, not plural

```suggestion
        <nav class="nav">
          <ul class="nav__list">
            <li class="nav__item">
              <a
                href="#apple"
                class="nav__link is-active"
```